content/en/blog/2023/sunsetting-opencensus.md

```diff
@@ -0,0 +1,91 @@
+OpenCensus and a migration story for existing users.
+
+To help ease the migration path, we have provided backward compatibility bridges
+in [Go][go shim], [Java][java shim], [Python][python shim][^shim-next-release],
+and [JavaScript][js shim][^shim-next-release]. Installing these bridges allows
+OpenCensus and OpenTelemetry instrumentation to smoothly interoperate, with all
+of your telemetry flowing out of OpenTelemetry exporters. This lets OpenCensus
+users incrementally transition all of their instrumentation from OpenCensus to
+OpenTelemetry, and finally remove OpenCensus libraries from their applications.
+
+While OpenTelemetry was never intended to be a strict superset of OpenCensus,
+most of the APIs and data models are compatible. Migration should be considered
+a "major version bump" and you may notice some changes in your telemetry.
+
+More details on what to expect and some suggested workflows for migration are
+outlined in the
+[OpenCensus Compatibility specification](/docs/reference/specification/compatibility/opencensus#migration-path)[^spec-next-release].
+
+## What to Expect After July 31st, 2023
+
+After July 31st, 2023, the OpenCensus project will no longer be maintained. This
+means that there will be no new features added to the project, and any security
+vulnerabilities that are found will not be patched.
+
+However, the OpenCensus repositories will remain archived on GitHub. This means
+users will still be able to download the OpenCensus code and use it in their
+projects. Existing releases of OpenCensus will remain available in public
+package repositories like NPM and PyPI. **We encourage all OpenCensus users to
+migrate to OpenTelemetry as soon as possible.**
+
+One exception to this is the
+[census-instrumentation/opencensus-python][][^python-timeline] repo.
+
+[go shim]:
+  https://github.com/open-telemetry/opentelemetry-go/tree/main/bridge/opencensus
+[java shim]:
+  https://github.com/open-telemetry/opentelemetry-java/tree/main/opencensus-shim
+[python shim]:
+  https://github.com/open-telemetry/opentelemetry-python/tree/main/shim/opentelemetry-opencensus-shim
+[js shim]:
+  https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-shim-opencensus
+[census-instrumentation/opencensus-python]:
+  https://github.com/census-instrumentation/opencensus-python
+
+[^python-timeline]:
+    A number of projects within this repository are still being used as
+    recommended production solutions. These projects will continue to be
+    maintained. Please reach out to the individual maintainers for details
+    regarding maintenance timeline, next steps for migration and general support
+    questions.
+
+[^shim-next-release]:
+    Python and JavaScript shim packages are currently unreleased but will be
+    available in future releases.
+
+[^spec-next-release]:
+    The OpenCensus Compatability specification is marked stable for the next
```

Comment:
Minor: consider adding a link to the spec.

Reply:
done